app/actions/getConfig.ts

```diff
@@ -1,34 +1,131 @@
 'use server';
+import { ServiceTitanConfig } from '@/lib/types/serviceTitan';
+import { ServiceToJobType } from '@/lib/types/serviceToJobType';
+import { StripeConfig } from '@/lib/types/stripe';
+import { CustomFields } from '@/lib/types/customFields';
+import { getSetting } from "@/lib/appSettings/getSetting";
+import pino from "pino";
 
-import { config } from '@/lib/config';
 
-export async function getServiceTypes() {
-  return config.serviceToJobTypes;
+// const logger = pino({ name: 'getAvailableTimeSlots' });
+const logger = pino({ name: 'getConfig' });
+
+export async function getServiceToJobTypes(): Promise<ServiceToJobType[]> {
+  const serviceToJobTypes = await getSetting('serviceToJobTypes');
```

Comment:
I'm thinking about how to strongly type these field names.

Maybe just a new TS type like:
```
type SettingName = "serviceToJobTypes" | "podium.locationId" // etc ...;
```

Then we can enforce that at on update/read at build time.

Bonus points.... I bet Zod has a way to enforce it a runtime.